EIPS/eip-1.md

```diff
@@ -58,11 +58,11 @@ Each status change is requested by the EIP author and reviewed by the EIP editor
   * :x: Last Call -- A request for Last Call status will be denied if material changes are still expected to be made to the draft. We hope that EIPs only enter Last Call once, so as to avoid unnecessary noise on the RSS feed.
 * **Last Call** -- This EIP will listed prominently on the https://eips.ethereum.org/ website (subscribe via RSS at [last-call.xml](/last-call.xml)).
   * :x: -- A Last Call which results in material changes or substantial unaddressed technical complaints will cause the EIP to revert to Draft.
-  * :arrow_right: Accepted (Core EIPs only) -- A successful Last Call without material changes or unaddressed technical complaints will become Accepted.
+  * :arrow_right: Accepted (Core EIPs only) -- This EIP is in the hands of the Ethereum client developers. 
   * :arrow_right: Final (Not core EIPs) -- A successful Last Call without material changes or unaddressed technical complaints will become Final.
```

Comment:
@Arachnid given your comment saying that a Core EIP can be final, does it mean that Core EIPs have an extra step over non core ones? Where Core EIPS go from `Last Call` to `Accepted` to `Final` and non core from `Last Call` to `Final` directly?

Reply:
@timbeiko I believe that's the current process, yes. We should probably eliminate 'Accepted' as a state entirely, though - since we're trying to remove dependency on ACD, and that data is encapsulated by Fork Meta EIPs anyway.

Reply:
Happy to do that, but I think we should make that change in another PR, once https://github.com/ethereum/EIPs/pull/1852#issuecomment-475663747 is merged. 

The purpose of this PR is to remove the conflicting definitions of `Accepted`. Removing `Accepted` altogether will take more work, so ideally I'd like to get this small change merged and see PR-1852 approved before attempting it.